plutus-tx-plugin/src/PlutusTx/Compiler/Expr.hs

```diff
@@ -487,7 +487,10 @@ compileExpr e = withContextM 2 (sdToTxt $ "Compiling expr:" GHC.<+> GHC.ppr e) $
                     GHC.$+$ (GHC.ppr $ GHC.realIdUnfolding n)
 
         -- arg can be a type here, in which case it's a type instantiation
-        l `GHC.App` GHC.Type t -> PIR.TyInst () <$> compileExpr l <*> compileTypeNorm t
+        l `GHC.App` GHC.Type t | GHC.isRuntimeRepKindedTy t -> withContextM 2 (sdToTxt $ "Ignoring 'LiftedRep:" GHC.<+> GHC.ppr t) $ compileExpr l -- throwSd CompilationError $ "Blabla"  -- compileExpr l
+
+        -- arg can be a type here, in which case it's a type instantiation
+        l `GHC.App` GHC.Type t -> withContextM 2 (sdToTxt $ "Applying type to expr:" GHC.<+> GHC.ppr (l, t)) $ PIR.TyInst () <$> compileExpr l <*> compileTypeNorm t
```

Comment:
```suggestion
        l `GHC.App` GHC.Type t -> withContextM 2 (sdToTxt $ "Applying expr to type:" GHC.<+> GHC.ppr (l, t)) $ PIR.TyInst () <$> compileExpr l <*> compileTypeNorm t
```

Reply:
Lol, thanks, I always get confused on this.